db/db_local/start_database.go

```diff
@@ -142,7 +142,7 @@ func StartDB(port int, listen StartListenType, invoker constants.Invoker) (start
 
 // startPostgresProcessAndSetPassword starts the postgres process and writes out the state file
 // after it is convinced that the process is started and is accepting connections
-func startPostgresProcessAndSetPassword(port int, listen StartListenType, invoker constants.Invoker) (e error) {
+func startPostgresProcessAndSetPassword(port int, listen StartListenType, invoker constants.Invoker) (datName string, e error) {
 	utils.LogTime("startPostgresProcess start")
 	defer utils.LogTime("startPostgresProcess end")
```

Comment:
I think this function should read as follows: (with error handling of course) 
```
func startPostgresAndSetPassword(port int, listen StartListenType, invoker constants.Invoker) (string, error) {
 
 process, err := startPostgreProcess(port, listen, invoker)

  databaseName, err = retrieveDatabaseName()

  err := setPassword()
}

```

}
```